components/vector-tekton-logs-collector/development/vector-helm-values.yaml

```diff
@@ -17,7 +17,8 @@ customConfig:
     kubernetes_logs:
       type: kubernetes_logs
       rotate_wait_secs: 5
-      glob_minimum_cooldown_ms: 15000
+      glob_minimum_cooldown_ms: 500
```

Comment:
Maybe do that 1s (1000)? IMO that'll be enough to detect the new logs, without stressing the system too much. I've already heard complains of vector (not our deployment) being too resource hungry.

Reply:
I use this reference while setting this: https://github.com/vectordotdev/vector/issues/7934#issuecomment-864972060

Reply:
OK. I was thinking most short lived taskruns I've seen are taking ~3s, so setting that value to 1s will be enough to not miss logs without over stressing the system. Anyway, we can start with that lower value and monitor the resource utilization and increase the value if needed.